UM/Application.py

```diff
@@ -348,4 +357,57 @@ def getInstallPrefix():
         else:
             return os.path.abspath(os.path.join(os.path.dirname(sys.executable), ".."))
 
+    def _mrproper(self, only_user = False):
+        path_list = []
+
+        if not only_user:
+            if Platform.isWindows():
+                os.path.join(os.path.expanduser(), )
+                dir_patterns = ["\\AppData\\Local\\%s" %self._application_name,
+                                ]
+                file_patterns = []
+            elif Platform.isOSX():
+                dir_patterns = ["Library/Application Support/%s" %self._application_name,
+                                ]
+                file_patterns = []
+            elif Platform.isLinux():
+                dir_patterns = ['.config/%s' %self._application_name,
+                                '.cache/%s' %self._application_name,
+                                '.local/share/%s' %self._application_name,
```

Comment:
I hope you're going to remove this, but we use double quotes for strings and `os.path.join` instead of path separators :wink:

Reply:
Hope this was/is a rhetorical statement 😛